### DISCUSSION TITLE
Where to host Django/Flask apps

### DISCUSSION THREAD

**Discussion by RemcoHalman** [EddieHubCommunity/support#135] [Q&A] [answered]
I am researching places to host my Django and Flask apps. Localhost works well for on location apps. I'm trying to get client based work online (websites/apis). Any tips?

**Comment by eddiejaoude** [comment]
@RemcoHalman I havent, but I shared this discussion in the `python` channel in our Discord, I know we have some python people there - hopefully they can add some ideas here

**Comment by Krishna-Ravi** [comment]
@RemcoHalman why don't you try out heroku.
It is a good hosting platform and the steps are also simple.

**Comment by mikeysan** [comment, marked as answer]
@Krishna-Ravi Has already suggested what I was going to say. I haven't used Django/Flask yet, but I do host a discord bot on Heroku for free. They have command-line tools to make it simple to work with.
You'll need to set up a free account with them, and then hopefully this might help.
[deploy python on heroku]https://devcenter.heroku.com/articles/deploying-python)

**Comment by AllanRegush** [comment]
I second Heroku. If your looking for something more for production consider setting up a server with digital ocean.

**Comment by RemcoHalman** [comment]
Does Heroku not become expensive when leaving the Free tier? Ill look into it 👌🏼Thnx

**Comment by ankushKun** [comment]
[Repl.it](https://repl.it/) is a pretty good service to host code, I am hosting my discord python bot over there, which uses Flask to make a webserver and keep the bot online.
Moreover it is free and has much simpler interface than heroku.
It can also give 99% uptime if you keep pinging the webserver, unlike heroku which gives only 550hr uptime permonth

**Comment by RemcoHalman** [comment]
Thanks for the input! Will look into Heroku and repl.it

**Comment by siddharthaweb** [comment]
looking for the best economic solution to deploy a flask application on the web, and also wants to know the deployment process. 
I am a beginner in python and flask, if you have any good structural suggestion (MVC in the flask with JWT token ) for API will be helpful.
